hbase-server/src/main/java/org/apache/hadoop/hbase/master/region/MasterRegionFactory.java

```diff
@@ -89,45 +82,8 @@
   private static final TableDescriptor TABLE_DESC = TableDescriptorBuilder.newBuilder(TABLE_NAME)
     .setColumnFamily(ColumnFamilyDescriptorBuilder.of(PROC_FAMILY)).build();
 
-  private final LocalRegion region;
-
-  private LocalStore(LocalRegion region) {
-    this.region = region;
-  }
-
-  public void update(UpdateLocalRegion action) throws IOException {
-    region.update(action);
-  }
-
-  public Result get(Get get) throws IOException {
-    return region.get(get);
-  }
-
-  public RegionScanner getScanner(Scan scan) throws IOException {
-    return region.getScanner(scan);
-  }
-
-  public void close(boolean abort) {
-    region.close(abort);
-  }
-
-  @VisibleForTesting
-  public FlushResult flush(boolean force) throws IOException {
-    return region.flush(force);
-  }
-
-  @VisibleForTesting
-  public void requestRollAll() {
-    region.requestRollAll();
-  }
-
-  @VisibleForTesting
-  public void waitUntilWalRollFinished() throws InterruptedException {
-    region.waitUntilWalRollFinished();
-  }
-
-  public static LocalStore create(Server server) throws IOException {
-    LocalRegionParams params = new LocalRegionParams().server(server)
```

Comment:
How comes we can drop these methods?

Reply:
We will return MasterRegion directly after this change, so we do not need these methods any more, users can use the methods in MasterRegion directly.

Reply:
👍

Reply:
ok